core/sphinx/geo/geo.go

```diff
@@ -0,0 +1,344 @@
+package geo
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/fxamacker/cbor/v2"
+	"golang.org/x/crypto/sha3"
+
+	"github.com/cloudflare/circl/kem"
+	kemschemes "github.com/cloudflare/circl/kem/schemes"
+
+	"github.com/katzenpost/katzenpost/core/crypto/nike"
+	"github.com/katzenpost/katzenpost/core/crypto/nike/schemes"
+	"github.com/katzenpost/katzenpost/core/sphinx/internal/crypto"
+)
+
+const (
+	NodeIDLength      = 32
+	RecipientIDLength = 32
+	SURBIDLength      = 16
```

Comment:
these 3 values are still also defined (with the same values as here) in `core/sphinx/constants.go`; presumably `constants.go` should be removed in favor of this?